src/main/java/com/appdynamics/extensions/webspheremq/metricscollector/QueueMetricsCollector.java

```diff
@@ -99,26 +104,10 @@ public void publishMetrics() throws TaskExecutionException {
 		}
 	}
 
-	private Map<String, WMQMetricOverride> getMetricsToReport(String command) {
-		Map<String, WMQMetricOverride> commandMetrics = Maps.newHashMap();
-		if (metrics == null || metrics.isEmpty()) {
-			logger.debug("There are no metrics configured for {}",command);
-			return commandMetrics;
-		}
-		for (Map.Entry<String, WMQMetricOverride> entry : metrics.entrySet()) {
-			String key = entry.getKey();
-			WMQMetricOverride wmqOverride = entry.getValue();
-			if (wmqOverride.getIbmCommand().equalsIgnoreCase(command)) {
-				commandMetrics.put(key, wmqOverride);
-			}
-		}
-		return commandMetrics;
-	}
-
-	protected void processPCFRequestAndPublishQMetrics(String queueGenericName, PCFMessage request, String command) throws IOException, MQDataException {
+	protected void processPCFRequestAndPublishQMetrics(MetricsCollectorContext context, String queueGenericName, PCFMessage request, String command) throws IOException, MQDataException {
```

Comment:
Hm.....   hi @breedx-splk ,   How come in my IntelliJ, this tells me that this method is not used?